paper/paper.md

```diff
@@ -145,21 +145,15 @@ _--documentation_: Instead of traversing the whole file, this feature annotates
 
 !!! Check the NeXus links! Use either nexusformat.org links, or reference the path in the actual(!) pynxtools repo noting that pynxtools/definitions is a github submodule and content (as well as content path) can change version to version.
 
-# NOMAD integration (schema, parser)
+# NOMAD integration
 
-While pynxtools works as a standalone tool using the command line, it can also be integrated directly into Research Data Management Systems (RDMS). Out of the box, the package functions as a plugin within the NOMAD platform, converting and parsing data from experiments. This enables experimental data in the NeXus format to be integrated into NOMAD's metadata model, making it searchable and interoperable with other data from theory and experiment. The plugin consists of several key components (so called entry points):
+While pynxtools works as a standalone tool using the command line, it can also be integrated directly into Research Data Management Systems (RDMS). Out of the box, the package functions as a plugin within the NOMAD platform, converting and parsing data from experiments. This enables data in the NeXus format to be integrated into NOMAD's metadata model, making it searchable and interoperable with other data from theory and experiment.
 
-- Schema Package: The NeXus (meta)data definitions are expressed in XML using the NeXus Definition Language (NXDL), which in turn is defined using XSD. pynxtools converts this representation and extends NOMAD's internal data schema (called __Metainfo__) with these domain-specific quantities. pynxtools also connects the NeXus vocabulary to existing base sections in NOMAD — reusable, standardized building blocks used to represent common scientific concepts. This connection enables interoperability between NeXus-defined concepts and other standardized representations in NOMAD, such as those for sample synthesis or theoretical calculations.
+The __DataConverter__ as described above is available in NOMAD, allowing users to convert their experimental data to NeXus through NOMAD's graphical interface. The __DataConverter__ class also handles NOMAD's electronic lab notebooks (ELNs), converting manual input data to NeXus.
```

Comment:
"The ... allowing users" > "This __DataConverter__ functionality of pynxtools in NOMAD allows users"

Reply:
The point that the manually inputted data are also part of the conversion does not come across but is important as one can therefore also search for these.